### DISCUSSION TITLE
Can I auto-sync helm chart to latest version?

### DISCUSSION THREAD

**Discussion by onuryartasi** [argoproj/argo-cd#5731] [Q&A] [answered]
Hello everyone,
I want to when I deploy helm chart to chartmusuem and ArgoCD sync to latest version. 
I try to range version eg. ~1.2.3 or wildcard patch 1.0.* but  I got semver errors.

![image](https://user-images.githubusercontent.com/16039143/110669003-f5b4ff00-81dc-11eb-921e-069ca1d93e6c.png)

**Comment by alexmt** [comment, marked as answer]
Can you please specify your Argo CD version? I suspect the was a bug that is already fixed in master. I will try to find which PR fixed it and probably cherry-pick change to affected version.

**Reply by onuryartasi** [reply]
Hi,
Thank you for quick response. I'm using argocd on minikube and kubernetes version is 1.20. I also  put the argocd version output below.
```json
{
    "Version": "v1.8.7+eb3d1fb",
    "BuildDate": "2021-03-03T07:13:34Z",
    "GitCommit": "eb3d1fb84b9b77cdffd70b14c4f949f1c64a9416",
    "GitTreeState": "clean",
    "GoVersion": "go1.14.12",
    "Compiler": "gc",
    "Platform": "linux/amd64",
    "KsonnetVersion": "v0.13.1",
    "KustomizeVersion": "v3.8.1 2020-07-16T00:58:46Z",
    "HelmVersion": "v3.4.1+gc4e7485",
    "KubectlVersion": "v1.17.8",
    "JsonnetVersion": "v0.17.0"
}
```

**Reply by onuryartasi** [reply]
I resolved problem. The explanation is here https://github.com/argoproj/argo-cd/issues/5740

**Comment by onuryartasi** [comment]
Hi @alexmt,
I'm adding screenshot when I push new chart to repository, argocd detect this version in repository  but cant auto-sync this version.
![Screen Shot 2021-03-11 at 13 38 28](https://user-images.githubusercontent.com/16039143/110775085-92c07800-826f-11eb-82b7-064057657d47.png)
 And below you can see latest version is 1.0.8
![Screen Shot 2021-03-11 at 13 38 57](https://user-images.githubusercontent.com/16039143/110775106-95bb6880-826f-11eb-9b1b-8a1399e51e0c.png)
